cmake/CPM.cmake

```diff
@@ -272,6 +272,33 @@ function(CPMFindPackage)
     endif()
   endif()
 
+  if(DEFINED CPM_ARGS_INCLUDE_IF)
+
+    # Filter out INCLUDE_IF from arguments
+    cpm_remove_one_value_arg(INCLUDE_IF ARGN ${ARGN})
```

Comment:
This function currently assumes that the user has always supplied a value for `INCLUDE_IF`.
That might be okay, but for CMake 3.15 and newer, we could also check if `INCLUDE_IF` is not in `CPM_ARGS_KEYWORDS_MISSING_VALUES` and otherwise inform this function about the fact that there is no value to be removed (see [cmake_parse_arguments docs](https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html).